.devcontainer/devcontainer.json

```diff
@@ -38,4 +38,4 @@
   "mounts": [
     "source=${localEnv:HOME}/repos,target=/home/node/repos,type=bind,consistency=cached"
   ]
-}
+}
```

Comment:
While we are modifying the instructions on development, can we have a brief note that you may have to change the remoteUser to root if the perms aren't working?